package.json

```
@@ -18,30 +18,32 @@
     "init": "node init.js"
   },
   "devDependencies": {
-    "@aws-cdk/assert": "1.106.1",
-    "@milltechfx/base-aws-stack": "^1.0.56",
+    "@aws-cdk/assert": "^1.111.0",
```

for importing any `@aws-cdk/*` dependencies let use fixed version - elsewhere we will have problems...
```suggestion
    "@aws-cdk/assert": "1.111.0",
```